examples/tfengine/generated/org_foundation/audit/variables.tf

```diff
@@ -13,9 +13,46 @@
 # limitations under the License.
 
 variable "org_id" {
-  type = string
+  type        = string
+  description = "ID of the organization to apply the configuration."
+  validation {
+    condition     = can(regex("^[0-9]{8,25}$", var.org_id))
+    error_message = "The org_id must be valid. Should have only numeric values with a length between 8 and 25 digits. See https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy to know how to get your organization id."
+  }
 }
 
 variable "auditors_group" {
-  type = string
+  type        = string
+  description = "This group will be granted viewer access to the audit log dataset and bucket as well as security reviewer permission on the root resource specified."
+}
+
+variable "filter" {
+  type        = list(string)
+  description = "Filters for log collection and export concatenated by \"OR\" operator. Refer to https://cloud.google.com/logging/docs/view/query-library for query syntax."
```

Comment:
Last PR i was talking with @MartinPetkov that there are cases when the description doesn't completely match with the variable. This is another example.

The original variable name at config level is `additional_filters`, and this is is `filter`, which is just a combination of defaults and the `additional`.

Nothing to change, just wanted to point it out.

Reply:
Might be solved now that #953 is done